rock_paper_scissors/rps.py

```diff
@@ -2,13 +2,73 @@
 
 import sys
 
+# n = players
+# we want to calculate the total number of combinations (inclusive) of moves for a given number of players
+
+# n=1
+# [rock]
+# [paper]
+# [scissors]
+
+# n = 2
+# [rock] [rock]
+# [rock] [paper]
+# [rock] [scissors]
+# [paper] [rock]
+# [paper] [paper]
+# [paper] [scissors]
+# [scissors] [rock]
+# [scissors] [paper]
+# [scissors] [scissors]
+
+# n = 3
+# [rock] [rock] [rock]
+# [rock] [rock] [paper]
+# [rock] [rock] [scissors]
+# [rock] [paper] [rock]
+# [rock] [paper] [paper]
+# [rock] [paper] [scissors]
+# [rock] [scissors] [rock]
+
+
+# [paper] [rock]
+# [paper] [paper]
+# [paper] [scissors]
+# [scissors] [rock]
+# [scissors] [paper]
+# [scissors] [scissors]
+
+moves = [["rock"], ["paper"], ["scissors"]]
+
+# plays = [[length of n]... all permutations of r/p/s]
 def rock_paper_scissors(n):
-  pass 
+    plays = list()
+
+    def play_combos(n, plays):
+        temp = list()
```

Comment:
Great to use temp variable, write a comment to explain how it is being used for the math behind this challenge